packages/itwin/grouping-mapping-widget/src/widget/components/GroupingMapping.scss

```diff
@@ -10,6 +10,11 @@
   width: 100%;
 }
 
+.iui-modal > .iui-modal-dialog {
```

Comment:
This will probably clash for other consumers who are might be using the same class name.